src/GraphNodeManagerViewExtension/Controls/SearchBoxControl.xaml

```diff
@@ -107,6 +112,28 @@
                             HorizontalAlignment="Stretch"
                             TextChanged="SearchTextBox_OnTextChanged"
                             Text="{Binding SearchText, UpdateSourceTrigger=PropertyChanged}"></TextBox>
+                        
+                        <!-- Clear Search -->
+
+                        <Button Grid.Column="1"
+                                    Margin="5 5 0 0"
+                                    PreviewMouseLeftButtonDown="OnSearchClearButtonClicked" 
+                                    Name="SearchClearButton"
+                                    Style="{StaticResource ButtonStyle1}"
+                                    Visibility="{Binding ElementName=SearchTextBox, 
+                                                         Path=Text, 
+                                                         Converter={StaticResource NonEmptyStringToCollapsedConverter}}"
+                                    Width="16px"
+                                    Height="16px"
+                                    HorizontalAlignment="Center"
+                                    VerticalAlignment="Center"
+                                    ToolTip="Clear search box .."
```

Comment:
Can we move this to resource string so that we wont have localization issues.

Reply:
Make sense! Looks like our localization team will need to make additional effort after 6/17..